netbsd/NetBSDProcess.c

```diff
@@ -0,0 +1,239 @@
+/*
+htop - NetBSDProcess.c
+(C) 2015 Hisham H. Muhammad
+(C) 2021 Santhosh Raju
+Released under the GNU GPLv2, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "NetBSDProcess.h"
```

Comment:
Should include platform path (relative to main directory path) -> `netbsd/NetBSDProcess.h`

Reply:
Should be fixed in f20c703